src/features/metadata-instance-editor/Instance.js

```diff
@@ -642,6 +650,7 @@ class Instance extends React.PureComponent<Props, State> {
                                         canEdit={isEditing}
                                         data={data}
                                         errors={errors}
+                                        templateFilters={templateFilters}
```

Comment:
Minor, but can we alphabetize these props? Same for the destructured object above?